jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/src/main/java/org/jbpm/workbench/cm/server/MockCaseManagementService.java

```diff
@@ -55,8 +59,10 @@
     private static final String CASE_DEFINITIONS_JSON = "case_definitions.json";
     private static final String CASE_MILESTONES_JSON = "case_milestones.json";
     private static final String CASE_STAGES_JSON = "case_stages.json";
+    private static final String CASE_ACTIONS_JSON = "case_actions.json";
 
     private static int commentIdGenerator = 0;
+    private static long actionIdLongenerator = 8;
```

Comment:
This is just a mock class so its fine, but you could use AtomicInteger and AtomicLong which are thread safe.